src/java/org/apache/cassandra/replication/MutationTrackingService.java

```diff
@@ -325,6 +363,57 @@ public boolean registerMutationCallback(ShortMutationId mutationId, IncomingMuta
         return incomingMutations.subscribe(mutationId, callback);
     }
 
+    public void executeTransfers(String keyspace, Set<SSTableReader> sstables, ConsistencyLevel cl)
+    {
+        shardLock.readLock().lock();
+        try
+        {
+            logger.info("Creating tracked bulk transfers for keyspace {} sstables {}", keyspace, sstables);
```

Comment:
```suggestion
            logger.info("Creating tracked bulk transfers for keyspace '{}' SSTables {}...", keyspace, sstables);
```
I'll try not to bike-shed on log messages too much...I promise...